### ISSUE TITLE
[Content group banner]: QA: No label is defined for the dialog

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9654]
### Description

Page Element:
The dialog we get after selecting CTA type video from Knobs

Issue Description:
No label for the dialog

Recommend Fix:
Provide A descriptive label. This issue is already opened for the lightbox component

User Impact:
When the screen reader is unable to identify a label, it’s difficult for users to understand the purpose of the dialog.

Tool:
Voice Over_iPhone
JAWS_Desktop

### Component(s) impacted

Content group banner

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-content-block-media--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
